content/en/Integrations/Beta/_index.md

```diff
@@ -8,10 +8,20 @@ description: >
 ---
 
 {{% pageinfo %}}
-<p>{{< beta-label >}}</p><p><a href="#submit-a-request">Request to enable a beta integration</a> for your next pentest to streamline your remediation workflows. Customize the configuration to suit your needs, and start pushing Cobalt findings to <b>Azure DevOps</b>, <b>Bitbucket</b>, or <b>ServiceNow</b>.</p><p><span style="background-color: #ECE6FA; padding: 2px;">Navigate to the <b>Integrations</b> page in the Cobalt app to get started.</span> {{< tier-feature-html>}}</p>
+<p>{{< beta-label >}}</p><p><a href="#submit-a-request">Request to enable a beta integration</a> for your next pentest to streamline your remediation workflows. Customize the configuration to suit your needs, and start pushing Cobalt findings to your preferred task management software.</p><p><span style="background-color: #ECE6FA; padding: 2px;">Navigate to the <b>Integrations</b> page in the Cobalt app to get started.</span> {{< tier-feature-html>}}</p>
 {{% /pageinfo %}}
 
-Beta integrations are fully functional connectors that we’ve developed using [Workato](https://www.workato.com/)—a workflow automation platform that enables greater flexibility to tailor our integrations to customers' unique workflows.
+Beta integrations are fully functional connectors that we've developed using [Workato](https://www.workato.com/)—a workflow automation platform that enables greater flexibility to tailor our integrations to customers' unique workflows. We support integrations with:
+
+- Asana
+- Azure DevOps (Boards)
+- Bitbucket Issues
+- Jira (Beta)
```

Comment:
Please remove "Jira (Beta)" from the "Request a Beta Integration". We've been piloting this for select customers who are blocked by our limited support for Jira custom fields, but are not making it available (for now at least) to most customers.